test/test-parse-module.cpp

```diff
@@ -797,6 +797,18 @@ TEST(test_parse, imported_variables_can_be_named_contextual_keywords) {
   }
 }
 
+TEST(test_parse, imported_modules_must_be_quoted) {
+  padded_string code(u8"import { test } from module;"sv);
+  SCOPED_TRACE(code);
```

Comment:
Nit: `SCOPED_TRACE` is unnecessary because the developer can look at the source code manually. I'd remove this line.

Reply:
Fixed